packages/modules/workflow-engine-redis/src/services/workflow-orchestrator.ts

```diff
@@ -689,6 +685,15 @@ export class WorkflowOrchestratorService {
       state,
     } = options
 
+    if (publish && !isFlowAsync) {
```

Comment:
thought: this might be obvious if you know the implementation details, but I read this if-statement as such "Publish message if publish is true and the flow is not asynchronous" which kind of contradicts the intention. Am I missing something?

Reply:
oih my bad, you are right 😓 for some reason I didnt see I ve put a negation there nice catch